### ISSUE TITLE
Disallow `Object.assign({}, ...)` by `prefer-spread`

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/eslint-plugin-unicorn#1065]
## Fail

```js
var replace = Object.assign({}, a, b);
```

## Pass

```js
const replace = {...a, ...b};
```

**Comment by sindresorhus** [comment]
That's already covered by https://eslint.org/docs/rules/prefer-object-spread We should document that in the rule docs.
